Delete_User.css

```diff
@@ -104,44 +128,77 @@ ul li:hover ul{
 	width:100%
 }
 
+/* Set up the footer */
```

Comment:
I make some change to the footer, l by adding margin and some update likes fonts, color.